repo/packages/M/mysql/0/config.json

```diff
@@ -0,0 +1,59 @@
+{
+    "type": "object",
+    "properties": {
+        "mysql": {
+            "description": "MySQL framework configuration properties",
+            "type": "object",
+            "properties": {
+                "framework-name": {
```

Comment:
this is the older way of managing service names.  It will result in a poor naming of the service.   you likely want to user the "service.name" instead.  see https://github.com/mesosphere/universe-1.7/blob/version-2.x/repo/packages/K/kafka/6/config.json#L8